File: PHP Excel Examples/CellsApiDemo/Row/getRows.php
<?php

use Spire\Cloud\Excel\Sdk\Api\CellsApi;
use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new CellsApi($configuration);

$name = "GetRows.xlsx";
$sheetName = "Sheet1";
$folder = "input";
$storage = null;
$result = $apiInstance->getRows($name, $sheetName, $folder, $storage);

File: Python Excel Examples/CellsApiDemo/row/getRows.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.cells_api import CellsApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.cells_api.CellsApi(configuration)
name = "GetRows_1.xlsx"
sheetName = "Sheet1"
storage = ""
folder = "/ExcelDocument/"
result = api.get_rows(name, sheet_name=sheetName, folder=folder, storage=storage)
